demo.scss

```diff
@@ -2,7 +2,7 @@
 @import "scss/functions";
 @import "scss/variables";
 
-header {
+header:not(.table-header) {
```

Comment:
Why do we need this? I know it's just for demo purposes but we should avoid any and every modifications that might affect our end result (potentially masking behaviors/appearance).
If it's only for demo I suggest using a new class, include the **.table-header** one and modifying that new class instead.

Reply:
I'm not sure if I follow your reasoning, but I suppose avoiding use of :not() when possible is a good rule of thumb. I've made the change. Hopefully, it addresses your concern.